src/main/java/racing/domain/RacingCars.java

```diff
@@ -18,4 +23,33 @@ public void moveCars() {
 	public List<Car> getRacingCars() {
 		return racingCars;
 	}
+
+	public List<Car> getWinnersOfRacing() {
+		List<Car> winners;
+
+		final int maxLocation;
+		try {
+			maxLocation = getMaxLocation();
+		} catch (NoSuchWinnerException exception) {
+			return Collections.emptyList();
+		}
+
+		winners = IntStream.range(0, racingCars.size())
```

Comment:
```suggestion
		final int maxLocation;
		try {
			maxLocation = getMaxLocation();
		} catch (NoSuchWinnerException exception) {
			return Collections.emptyList();
		}

		List<Car> winners = IntStream.range(0, racingCars.size())
```
지역변수를 미리선언해둘 필요가 없을 것 같아요. 조금 불편할 수 있겠지만 다음 미션은 모든 지역변수에 final 키워드를 붙이고, 꼭 필요할 때만 final 키워드를 제거해서 최대한 재할당하지 않고 코딩하는 습관을 길러보는 것도 좋을 것 같습니다 😄

Reply:
final을 붙이는 습관을 들이면 객체가 수정되는 경우를 좀 더 막을 수 있는 좋은 방법이 되겠네요! 다음 미션 때 꼭 실천해볼게요 👍